packages/xchain-thorchain/src/client.ts

```diff
@@ -613,6 +631,32 @@ class Client extends BaseXChainClient implements ThorchainClient, XChainClient {
       return getDefaultFees()
     }
   }
+
+  private async determineGasLmit({
+    txBody,
+    privKey,
+    multiplier,
+    fallbackGasLimit,
+  }: {
+    txBody: proto.cosmos.tx.v1beta1.TxBody
+    privKey: proto.cosmos.crypto.secp256k1.PrivKey
+    multiplier: number
+    fallbackGasLimit: string
+  }): Promise<string> {
+    try {
+      const gas_expected = await getGasExpectedForTx({ cosmosSDKClient: this.getCosmosClient(), txBody, privKey })
+      if (gas_expected === undefined) {
+        throw new Error('could not estimate gas limit required for TX')
+      }
+      const gasLimit = cosmosclient.Long.fromString(gas_expected).multiply(multiplier).toString()
```

Comment:
^ This could be already done in `util.getGasExpectedForTx`. And by removing the fallback (see another comment below), determineGasLmit can be removed from client.

Reply:
agreed, if I remove fallback gas limits, this entire method can be removed...so i'll wait until you reply above ;-)